### ISSUE TITLE
BigQuery connector requires `keyfile_creds`

### ISSUE THREAD

**Issue by seanpreston** [ethyca/fidesops#1333]

**Comment by eastandwestwind** [comment]
@seanpreston yes, BigQuery `keyfile_creds` is unlike other connector secrets in that it takes the form of a json object. More info here- https://ethyca.github.io/fidesops/guides/database_connectors/#google-bigquery

So we could use a larger input field in the UI, or a file upload.
